example_datascience/__main__.py

```diff
@@ -1,4 +1,220 @@
 import sys
+import pandas as pd
+import numpy as np
+from pycaret.classification import ClassificationExperiment, create_model, check_fairness
```

Comment:
Pycaret only supports python 3.9, 3.10, 3.11, so that may become a problem here for Python 3.12 users? (I coudn't run the code because of that)

Reply:
Yes that is true, hence i bumped the .python_version down to 3.11